config/sync/core.entity_view_display.node.stanford_publication.stanford_card.yml

```diff
@@ -42,8 +42,6 @@ third_party_settings:
           field_templates: default
           variant: default
     regions:
-      card_super_headline:
-        - citation_type
```

Comment:
This should still be there. is there a reason to remove it?

Reply:
No it should be there. I'm not sure why it isn't. I'll close this request.